docs/components/component/_index.md

```diff
@@ -6,9 +6,8 @@ weight: 10
 type: "docs"
 description: "A NAME is a ... description of what the component is."
 tags: ["camera", "components"]
-icon: "/icons/components.png"
-# icon: "path/to/svg/file"
-images: ["/icons/components.png"]
+# icon: true
+# images: ["/icons/components.svg"]
```

Comment:
Why don't we want an icon for this page?

Reply:
I mean it's a template and the svg doesn't exist, I can use the png though.